code/game/objects/items.dm

```diff
@@ -1023,6 +1023,11 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 
 /obj/item/proc/on_mob_death(mob/living/L, gibbed)
 
+/obj/item/proc/add_juice()
+	if(reagents)
+		return 1
+	return 0
```

Comment:
This seems incredibly weird, what does the return value mean and what does this proc actually do? If this is a stub for subtype implementation then shouldn't it always return false to indicate that its not actually doing anything?

Reply:
My bad, I moved the add_juice proc down from /obj/item/grown/ during testing.

I've moved it back up. Newfood doesnt really affect that system. I just forgot to fix it. Thanks for reminding me

Reply:
reverted in b0ca606